src/v/pandaproxy/schema_registry/storage.h

```diff
@@ -1287,7 +1292,13 @@ struct consume_to_store {
         try {
             vlog(plog.debug, "Applying: {}", key);
             if (!val) {
-                co_await _store.clear_compatibility(*key.sub);
+                co_await _store.clear_compatibility(
+                  seq_marker{
+                    .seq = key.seq,
+                    .node = key.node,
+                    .version{invalid_schema_version}, // Not applicable
+                    .key_type = seq_marker_key_type::config},
+                  *key.sub);
             } else if (key.sub) {
```

Comment:
it looks like key.sub is unconditionally dereferenced before its checked if it contains a value. is that happening somewhere else?